confluent_platform/metadata.csv

```diff
@@ -0,0 +1,316 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+kafka.network.request_channel.request_queue_size,gauge,,,,Size of the request queue. A congested request queue will not be able to process incoming or outgoing requests,0,confluent_platform,net req channel req queue size
+kafka.network.socket_server.network_processor_avg_idle_percent,gauge,,fraction,,Average fraction of time the network processor threads are idle. Values are between 0 (all resources are used) and 1 (all resources are available),0,confluent_platform,net sock net proc avg idle percent
```

Comment:
Thanks for using the `fraction` unit here. That being said, I would remove the part where you say that the values are between 0 and 1 because that won't be the case once in Datadog. (I mean they will be stored as a number between 0 and 1 but they will always appear as between 0 and 100 even when creating monitors)

Reply:
Sure, will update.